README.md

```diff
@@ -25,15 +25,15 @@ It contains a bare bones theme and must use plugin for you to base your developm
   ],
 ```
 6. To build plugins/themes simply run `npm install` at the root and `npm run [build|start|watch]` and npm will automatically build all themes and plugins.
-7. `npm workspaces` do not have the ability to run scripts from multiple packages in parrallel. Because of that we use the `npm-run-all` package and we define specific scripts in `package.json` so you will need to update the `watch:*` scripts in `package.json` and replace `tenup-theme` and `tenup-plugin` with the actual package names.
+7. `npm workspaces` do not have the ability to run scripts from multiple packages in parallel. Because of that we use the `npm-run-all` package and we define specific scripts in `package.json` so you will need to update the `watch:*` scripts in `package.json` and replace `tenup-theme` and `tenup-plugin` with the actual package names.
 
 ```json
 	"watch:theme": "npm run watch -w=tenup-theme",
 	"watch:plugin": "npm run watch -w=tenup-plugin",
 	"watch": "run-s watch:theme watch:plugin",
 ```
-7. To add npm dependencies to your theme and/or plugins add the `-w=package-name` flag to the `npm install` command. E.g: `npm install --save prop-types -w=tenup-plugin` **DO NOT RUN** `npm install` inside an individual workspace/package. Always run the from the root folder.
-8. If you're building Gutenberg blocks and importing `@wordpress/*` packages, **you do not** need to manually install them as `10up-toolkit` will handle these packages properly.
+7. To add npm dependencies to your theme and/or plugins add the `-w=package-name` flag to the `npm install` command. E.g: `npm install --save prop-types -w=tenup-plugin`. **DO NOT RUN** `npm install` inside an individual workspace/package. Always run it from the root folder.
+8. If you're building Gutenberg blocks and importing `@wordpress/*` packages, **you do not** need to manually install them as `10up-toolkit` will handle these packages properly. However, it might still be a good idea for intellisense and code completion.
```

Comment:
Yup, I personally like having those packages installed locally for code completion in VS Code, but I have also found that other engineers are confused to see them in package.json and this can lead to some incorrect assumptions when attempting to debug editor issues, which can waste (e.g. folks will think they need to update the `@wordpress/*` dependencies , when these packages in fact aren't bundled into the code). If these packages _are_ installed, though, I agree they belong in devDependencies as they're for local dev only (see my related suggestion at https://github.com/10up/wp-scaffold/issues/138)

Reply:
The point of the change was to not sound as harsh and rather give the option. You don’t need to but it’s not harmful